proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -121,10 +121,9 @@ message Validator {
       [(gogoproto.nullable) = false, (amino.dont_omitempty) = true, (gogoproto.stdtime) = true];
   // commission defines the commission parameters.
   Commission commission = 10 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
-  // min_self_delegation is the validator's self declared minimum self delegation.
-  //
-  // Since: cosmos-sdk 0.46
+  // Deprecated: This field has been deprecated with LSM in favor of the validator bond
   string min_self_delegation = 11 [
```

Comment:
I wonder if it would be appropriate to remove min_self_delegation here...

Or do we carry around the deprecated field into 0.51

Reply:
This is still not deprecated in SDK main. Not sure what's the plan for it. cc @tac0turtle @alexanderbez

Reply:
In general, we want to avoid removing fields and instead deprecate them via the `deprecate = true` flag